src/pods/toolbar/toolbar.pod.tsx

```diff
@@ -31,14 +31,68 @@ export const ToolbarPod: React.FC = () => {
       />
     );
   };
-
   return (
     <div className={classes.container}>
-      <button onClick={zoomIn}>Zoom in</button>
-      <button onClick={zoomOut}>Zoom out</button>
-      <button onClick={handleRelationClick}>Relation</button>
-      <button onClick={handleEditTableClick}>Edit Table</button>
-      <button onClick={handleCanvasSettings}>Canvas Settings</button>
+      <h1>Toolbar</h1>
+      <div className={classes.buttonContent}>
+        <button className={classes.buttonZoomIn} onClick={zoomIn}>
+          <svg
```

Comment:
Maybe we vould enclose this in a component that we just call

Reply:
I would create a generic button where you pass as props:
  - The Icon component that you want to use.
  - The text that will be displayed below the icon.
  - The call back that needs to be called.

Something like:

_./src/pods/toolbar/components/toolbar-button.tsx_

```tsx
interface Props {
   icon: React.ReactNode;
   label : string;
   onClick : () => void;
}
export const ToolbarButton : React.FC<Props> = props => {
// Your core here...
}
```

Then replace the buttons in this file with the ToolbarButton